include/deal.II/dofs/dof_renumbering.h

```diff
@@ -657,6 +657,15 @@ namespace DoFRenumbering
   void
   block_wise (DoFHandler<dim,spacedim> &dof_handler);
 
+  /**
+   * Sort the degrees of freedom by vector block. It does the same thing as the
+   * above function, only that it does this for one single level of a multi-
+   * level discretization. The non-multigrid part of the the DoFHandler is not
```

Comment:
don't break words here -- doxygen does not preserve linebreaks in creating output.